### DISCUSSION TITLE
Configuring EJBCA Community Docker Container for Persistent Use on AWS

### DISCUSSION THREAD

**Discussion by Babacar98** [Keyfactor/ejbca-ce#699] [Ideas]
I am trying to install an EJBCA Community instance on AWS using Docker on my physical machine. When I launch the instance with Docker and retrieve the superadmin certificate, which I then import into my browser, everything seems to go well. However, when I restart the Docker terminal, I lose everything and get a new superadmin certificate with a new superadministrator password. I do not want my Docker container to be ephemeral on my AWS EC2 instance. How can I keep the instance to connect it to a database on AWS without losing the container?

**Comment by aloeenmae** [comment]
@Babacar98 you need to use an external database for this: https://hub.docker.com/r/keyfactor/ejbca-ce  "External database section" or https://github.com/Keyfactor/keyfactorcommunity/blob/main/deployment-examples/docker-engine/ejbca-ce-three-level-architecture/README.md
